docs/serverless/rules/detections-logsdb-impact.asciidoc

```diff
@@ -0,0 +1,63 @@
+[[detections-logsdb-index-mode-impact]]
+= The impact of usage of logsDB index mode with {sec-serverless}
+
+LogsDB is enabled by default for Elastic serverless.
+
+With logsDB index mode, the original `_source` field is not stored in the index but can be reconstructed using {ref}/mapping-source-field.html#synthetic-source[synthetic `_source`].
+
+During the reconstruction of the `_source`, {ref}/mapping-source-field.html#synthetic-source-modifications[modifications compared] to the original data are possible. There could however happen to be a mismatch between how users expect field values to be formatted.
+
+What does it mean for you?
+
+[discrete]
+[[logsdb-alerts]]
+== Alerts
+
+During alert creation, the `_source` event is copied over into the alert in order to keep reference to the original data. When the logsDB index mode is applied, the `_source` event stored in the alert will be an event reconstructed using synthetic `_source`.
+
+If you are switching to using logsDB index mode, the `_source` field stored in the alert might look different in certain situations:
+
+* {ref}/mapping-source-field.html#synthetic-source-modifications-leaf-arrays[Arrays can be reconstructed differently or deduplicated]
+* {ref}/mapping-source-field.html#synthetic-source-modifications-field-names[Field names] 
+
+Alerts produced by the following rule types could be affected:
+
+* Custom query
+* Event correlation (non-sequence only)
+* Non-aggregate rule types
```

Comment:
Will need to specify what these are.

Reply:
Addressed in [85a24ab](https://github.com/elastic/security-docs/pull/6221/commits/85a24ab282bcf14c98cdd09dcaead40db78f9b0b).